tshark-4.4.yaml

```
@@ -45,6 +47,68 @@ pipeline:
   - uses: strip
 
 subpackages:
+  - name: ${{package.name}}-libs
+    dependencies:
+      provides:
+        - tshark-libs=${{package.full-version}}
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/lib
+          mv "${{targets.destdir}}"/usr/lib/* "${{targets.subpkgdir}}"/usr/lib
+    description: ${{package.name}} libraries
+    test:
+      pipeline:
+        - uses: test/tw/ldd-check
+
+  - name: editcap
+    description: Edit and/or translate the format of capture files
+    dependencies:
+      runtime:
+        - ${{package.name}}-libs
+    pipeline:
+      - runs: |
+          install -Dm755 "${{targets.destdir}}/usr/bin/editcap" "${{targets.contextdir}}/usr/bin/editcap"
+    test:
+      environment:
+        contents:
+          packages:
+            - tcpdump
+            - curl
+      pipeline:
+        - uses: test/tw/ldd-check
+        - name: Verify installation
+          runs: |
+            editcap --help
+            editcap --version
+        - name: Download a sample .pcap file
+          runs: |
+            curl https://raw.githubusercontent.com/wireshark/wireshark/master/test/captures/http-ooo2.pcap -o /tmp/test.pcap
+        - name: Verify packet split
+          runs: |
+            cd /tmp
+            no_of_packets=$(tcpdump -r test.pcap | wc -l)
+            # splitting each packet into its own file
+            editcap -c 1 test.pcap split.pcap
+            no_of_files=$(ls -1 split* | wc -l)
```

```suggestion
            no_of_files=$(find -type f -iname 'split*' | wc -l)
```
`ls`'s output is inconsistent depending on the implementation that we are dealing with, to make sure this wont blow up we can use `gnufind` instead (see https://unix.stackexchange.com/questions/128985/why-not-parse-ls-and-what-to-do-instead)